Musician Module/Controls/Instrument/Bass/BassSoundRepository.cs

```diff
@@ -0,0 +1,61 @@
+            // High Octave
+            {$"{GuildWarsControls.WeaponSkill1}{FluteNote.Octaves.High}", "C2"},
+            {$"{GuildWarsControls.WeaponSkill2}{FluteNote.Octaves.High}", "D2"},
+            {$"{GuildWarsControls.WeaponSkill3}{FluteNote.Octaves.High}", "E2"},
+            {$"{GuildWarsControls.WeaponSkill4}{FluteNote.Octaves.High}", "F2"},
+            {$"{GuildWarsControls.WeaponSkill5}{FluteNote.Octaves.High}", "G2"},
+            {$"{GuildWarsControls.HealingSkill}{FluteNote.Octaves.High}", "A2"},
+            {$"{GuildWarsControls.UtilitySkill1}{FluteNote.Octaves.High}", "B2"},
+            {$"{GuildWarsControls.UtilitySkill2}{FluteNote.Octaves.High}", "C3"}
+        };
+
+        private static readonly Dictionary<string, CachedSound> Sound = new Dictionary<string, CachedSound>
+        {
+            {"C1", new CachedSound(new AutoDisposeFileReader(new VorbisWaveReader(GameService.Content.GetFile(@"instruments\Bass\C1.ogg"))))},
+            {"D1", new CachedSound(new AutoDisposeFileReader(new VorbisWaveReader(GameService.Content.GetFile(@"instruments\Bass\D1.ogg"))))},
+            {"E1", new CachedSound(new AutoDisposeFileReader(new VorbisWaveReader(GameService.Content.GetFile(@"instruments\Bass\E1.ogg"))))},
+            {"F1", new CachedSound(new AutoDisposeFileReader(new VorbisWaveReader(GameService.Content.GetFile(@"instruments\Bass\F1.ogg"))))},
+            {"G1", new CachedSound(new AutoDisposeFileReader(new VorbisWaveReader(GameService.Content.GetFile(@"instruments\Bass\G1.ogg"))))},
+            {"A1", new CachedSound(new AutoDisposeFileReader(new VorbisWaveReader(GameService.Content.GetFile(@"instruments\Bass\A1.ogg"))))},
+            {"B1", new CachedSound(new AutoDisposeFileReader(new VorbisWaveReader(GameService.Content.GetFile(@"instruments\Bass\B1.ogg"))))},
+            {"C2", new CachedSound(new AutoDisposeFileReader(new VorbisWaveReader(GameService.Content.GetFile(@"instruments\Bass\C2.ogg"))))},
+            {"D2", new CachedSound(new AutoDisposeFileReader(new VorbisWaveReader(GameService.Content.GetFile(@"instruments\Bass\D2.ogg"))))},
+            {"E2", new CachedSound(new AutoDisposeFileReader(new VorbisWaveReader(GameService.Content.GetFile(@"instruments\Bass\E2.ogg"))))},
+            {"F2", new CachedSound(new AutoDisposeFileReader(new VorbisWaveReader(GameService.Content.GetFile(@"instruments\Bass\F2.ogg"))))},
+            {"G2", new CachedSound(new AutoDisposeFileReader(new VorbisWaveReader(GameService.Content.GetFile(@"instruments\Bass\G2.ogg"))))},
+            {"A2", new CachedSound(new AutoDisposeFileReader(new VorbisWaveReader(GameService.Content.GetFile(@"instruments\Bass\A2.ogg"))))},
+            {"B2", new CachedSound(new AutoDisposeFileReader(new VorbisWaveReader(GameService.Content.GetFile(@"instruments\Bass\B2.ogg"))))},
+            {"C3", new CachedSound(new AutoDisposeFileReader(new VorbisWaveReader(GameService.Content.GetFile(@"instruments\Bass\C3.ogg"))))}
```

Comment:
This is something I may need to modify on my end.  A new `ContentsManager` is passed to modules now that allows them to read the contents of their own ref.  I don't think it has a `GetFile`, function, yet, though.  Without it, once the sounds are moved out of the Blish HUD ref, these calls won't work anymore since the `ContentService` only loads files from the main Blish HUD ref.  There is `GetSound`, which you could try, but I have read that MonoGame's sound services aren't particularly robust, so it may not cover your needs.  https://github.com/blish-hud/Blish-HUD/blob/moduler-modules-rework/Blish%20HUD/Modules/Managers/ContentsManager.cs

Let me add a `GetFile` function to the manager so that you can still use those files in the NAudio dependency.  If we find that the MonoGame `SoundEffect` type can handle the audio well enough, it may be worth switching to reduce the number of dependencies needed, but at least for now, it's probably fine.

Reply:
I've added `GetFileStream` in the `ContentsManager` now.  That should allow you to load those sounds now, without issue.  https://github.com/blish-hud/Blish-HUD/blob/3a2c042ddd0abcc356753f7e24b942d496745db3/Blish%20HUD/Modules/Managers/ContentsManager.cs#L108-L114